tutorials/intro/Intro_Tutorial_1.ipynb

```diff
@@ -144,41 +195,208 @@
+    "        if number_of_people(s) <= 5:\n",
+    "            if i % 10 == 8:\n",
+    "                dev_sents.add(s)\n",
+    "            elif i % 10 == 9:\n",
+    "                test_sents.add(s)\n",
+    "            else:\n",
+    "                train_sents.add(s)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we'll apply the candidate extractor to the three sets of sentences. The results will be persisted in the database backend."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Clearing existing...\n",
+      "Running UDF...\n",
+      "[========================================] 100%%\n",
+      "\n",
+      "('Number of candidates:', 22195)\n",
+      "Clearing existing...\n",
+      "Running UDF...\n",
+      "[========================================] 100%%\n",
+      "\n",
+      "('Number of candidates:', 2796)\n",
+      "Clearing existing...\n",
+      "Running UDF...\n",
+      "[========================================] 100%%\n",
+      "\n",
+      "('Number of candidates:', 2697)\n",
+      "CPU times: user 6min 38s, sys: 1.63 s, total: 6min 39s\n",
+      "Wall time: 6min 39s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "for i, sents in enumerate([train_sents, dev_sents, test_sents]):\n",
+    "    cand_extractor.apply(sents, split=i)\n",
+    "    print(\"Number of candidates:\", session.query(Spouse).filter(Spouse.split == i).count())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading Gold Labels\n",
+    "\n",
+    "Finally, we'll load gold labels for development and evaluation. Even though Snorkel is designed to create labels for data, we still use gold labels to evaluate the quality of our models. Fortunately, we need far less labeled data to _evaluate_ a model than to _train_ it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AnnotatorLabels created: 2796\n",
+      "AnnotatorLabels created: 2697\n",
+      "CPU times: user 1min 41s, sys: 213 ms, total: 1min 42s\n",
+      "Wall time: 1min 42s\n"
+     ]
+    }
+   ],
+   "source": [
+    "from util import load_external_labels\n",
     "\n",
-    "Note that any time we execute a `UDFRunner` like `CorpusParser`, we can also execute it in parallel by running e.g.:\n",
-    "```python\n",
-    "corpus_parser.apply(doc_preprocessor, parallelism=20)\n",
-    "```\n",
-    "**Note, however, that this parallel execution will not work with SQLite, the database system used by default**; you will need to use e.g. Postgres for this!"
+    "%time load_external_labels(session, Spouse, annotator_name='gold')"
```

Comment:
This still takes about 1.5 min for me, probably slower on many computers; does that sound correct?